### PR TITLE
Update to Angular 18 and standalone API

### PR DESCRIPTION
**Pull request by boeckMt**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/dlr-eoc/ukis-frontend-libraries/blob/main/CONTRIBUTING.md#-commit-message-guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)
- [x] The Project is [building and passes all tests](https://github.com/dlr-eoc/ukis-frontend-libraries/blob/main/DEVELOPMENT.md#further-you-can-test-and-build-locally) without errors

## PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?
- Angular has been updated to version `^18.2.8`
- Components have been converted to the new [standalone API](https://angular.dev/guide/components/importing#standalone-components) from Angular.
- Component templates have been converted to the new [control flow](https://angular.dev/guide/templates/control-flow) from Angular.
- Some styles have been fixed for the clr v17 update that were not correct.
- Fixed icon direction of cds icons
- Some dependencies have been updated

## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

- [BREAKING CHANGE: update to angular 18](https://github.com/dlr-eoc/ukis-frontend-libraries/commit/6f806fde1f49d4d039d8c4221f16da8b5bbbead0)
- BREAKING CHANGE: refactor * to standalone
  - 4c75c99f83d1d39e24f3ffedee3ed4992822d61a
  - 0f3ba73c4de8e2672e42cef17b523a8826d0ba8c
  - 4a79d5f63a8f2e608ad353455f3173d9028a63d1
  - 194fb9aeae1728008d399d9f0bb4d5886167108b
  - ae656deec9dd048324f02ce36cc2023b2c1a4959
  - 9ced053dac2cfb35e678c5fe1622b3ba0db80a9a
  - e7b4a4f0fe0352812c7dfe1964f23c3b5e9d3d18
  - 5ce22fe012dde533376a714aa738d734bbc011f2
  - 59240e25150d94020ac0f8974fcb3540922f0a73
  - ac48c2dd83f12df07cd45b116e3360b64920df2e
  - 53ba3a644b931138c2e2385e62c7bae5bd4b5bfa
  - 44ac68b6baaef069e7816aa47520667717bf830f
  - 6ea37b8cbbf2b9f858a593b6e6598bc4f9e90ab6

## Other information
Is it part of one/more [packages](https://github.com/dlr-eoc/ukis-frontend-libraries/packages) and which?

- A pre-release version has been created https://github.com/dlr-eoc/ukis-frontend-libraries/releases/tag/v15.0.0-next.1

### CODE REVIEW

**Comment by boeckMt** [comment]
@lucas-angermann this [79ee484](https://github.com/dlr-eoc/ukis-frontend-libraries/pull/247/commits/79ee48404caa06dcc15097b34e1c219dad72e816) was the cause of the build dist errors.
Because of these imports, we had multiple different instances of the service that were causing these problems.

**Review by boeckMt — COMMENTED** [review]
A you found another one, thanks for the fix.
I think this happened through the automatic migration to the standalone API.

**Comment by lucas-angermann** [comment]
Yes, the bug prevented the 3D terrain to load in the local build. Now it works again. As far as I can see, the remaining demo apps work as usal.

File: TUTORIALS.md
Comment (lucas-angermann) [inline TUTORIALS.md:1]: We should check the tutorial after the PR with a fresh installation and also look at the map-cesium readme.

**Review by lucas-angermann — APPROVED** [review]
Thank you for the changes and fixes!

**Pull request merged**